project/GraalVM.scala

```diff
@@ -51,27 +51,35 @@ object GraalVM {
     * into engine distribution build, so we have to maintain these manually.
     */
 
-  val pythonPkgs = Seq(
-    "org.graalvm.python"   % "python-language"    % version,
-    "org.graalvm.python"   % "python-resources"   % version,
-    "org.bouncycastle"     % "bcutil-jdk18on"     % "1.76",
-    "org.bouncycastle"     % "bcpkix-jdk18on"     % "1.76",
-    "org.bouncycastle"     % "bcprov-jdk18on"     % "1.76",
-    "org.graalvm.llvm"     % "llvm-api"           % version,
-    "org.graalvm.truffle"  % "truffle-nfi"        % version,
-    "org.graalvm.truffle"  % "truffle-nfi-libffi" % version,
-    "org.graalvm.regex"    % "regex"              % version,
-    "org.graalvm.tools"    % "profiler-tool"      % version,
-    "org.graalvm.shadowed" % "json"               % version,
-    "org.graalvm.shadowed" % "icu4j"              % version,
-    "org.graalvm.shadowed" % "xz"                 % version
-  )
+  val pythonPkgs = if ("espresso".equals(System.getenv("ENSO_JAVA"))) {
```

Comment:
One of the mistakes I made when writing this code was that I have not specified this field as `private`. All of `pythonPkgs`, `jsPkgs`, and `espressoPkgs` can and *should* be private. I don't like the `if ("espresso"...)` condition repeated three times in this file. You can have this condition only on `langsPkgs` field that is actually public and is used from various projects in `build.sbt`.

Reply:
Done in 2e3b0ed